### PR TITLE
🐛 Source Google Analytics: Fix issue with custom error and simplify configuration

### LINKED ISSUE #7850: Simplify configuration for Google Analytics
## Tell us about the problem you're trying to solve
A user setting up Google Analytics has confusion on the configuration.

## Describe the solution you’d like

Google Analytics
The setup fields should be listed in this order after selecting Google Analytics as the source:

1. View ID - this should not be a obfuscated field as it's not a password.
2. Capitalize "Credentials" w/ Authenticate button (the user should see this first, click and authenticate, and then move on to the rest of the fields)
3. Make "Credentials" the first input
4. Start Date should come after "credentials" (since this is a customization setting, it should be after authentication).
4. Custom Reports can be relabelled as Custom Reports (optional) to show the user that data can be passed without entering anything here.
5. "window_in_days" should be given the title "Batch size (days). 
6. Create a ticket to investigate the implications of removing the `window_in_days` parameter entirely, as it is asking the user for information on an implementation detail which is an anti-pattern per the [UX handbook](https://docs.airbyte.io/connector-development/ux-handbook#input-configuration). Place this ticket on the backlog. My opinion is that we should remove the field and always use 90.

### LINKED ISSUE #8049: Google Analytics issue with custom reports if data does not exist
## Enviroment
- **Airbyte version**: 0.31.0-alpha
- **OS Version / Instance**: AWS EC2
- **Deployment**: Docker
- **Source Connector and version**: Google Analytics 0.1.9
- **Destination Connector and version**: Postgres
- **Severity**: Medium
- **Step where error happened**: Syncing data

## Current Behavior
When adding the following custom reports to the Google analytics source

`[{"name":"audience_overview","dimensions":["ga:date"],"metrics":["ga:users","ga:newUsers","ga:sessions","ga:sessionsPerUser","ga:pageviews","ga:pageviewsPerSession","avgSessionDuration","ga:bounceRate"]},{"name":"campaign_performance","dimensions":["ga:date","ga:campaign"],"metrics":["ga:users","ga:newUsers","ga:sessions","ga:bounceRate","ga:pageviewsPerSession","ga:goalConversionRateAll","ga:goalCompletionsAll","ga:goalValueAll"]},{"name":"channel_traffic","dimensions":["ga:date","ga:channelGrouping"],"metrics":["ga:users","ga:newUsers","ga:sessions","ga:percentNewSessions","ga:bounceRate","ga:pageviewsPerSession","ga:avgSessionDuration","ga:goalConversionRateAll","ga:goalCompletionsAll","ga:goalValueAll"]},{"name":"operating_system_report","dimensions":["ga:date","ga:operatingSystem","ga:browser"],"metrics":["ga:users","ga:newUsers","ga:sessions","ga:bounceRate","ga:pageviewsPerSession","ga:avgSessionDuration","ga:goalConversionRateAll","ga:goalCompletionsAll","ga:goalValueAll"]},{"name":"social_media_acquisitions","dimensions":["ga:date","ga:socialNetwork"],"metrics":["ga:sessions","ga:percentNewSessions","ga:newUsers","ga:bounceRate","ga:pageviews","ga:pageviewsPerSession","ga:avgSessionDuration","ga:transactions","ga:transactionsPerSession","ga:transactionRevenue"]},{"name":"traffic","dimensions":["ga:date","ga:pageTitle"],"metrics":["ga:users","ga:pageValue", "ga:entrances","ga:pageviews","ga:uniquePageviews","ga:avgTimeOnPage","ga:exitRate","ga:bounceRate"]},{"name":"adwords_campaigns","dimensions":["ga:date","ga:adwordsCampaignID"],"metrics":["ga:adlicks","ga:adCost","ga:CPC","ga:users","ga:sessions","ga:bounceRate","ga:goalConversionRateAll","ga:goalCompletionsAll","ga:goalValueAll"]}]`

We get an error message in the logs that the streams for some of this data do not exist. This causes the whole connector to fail.

## Expected Behavior
If these streams don't exist in the current users account the either two scenarios should happen:

1. (Preferred) Create an empty table with the desired columns of the report so if data is created then it will be synced to the database
2. Skip the custom and carry on to the next report

## Logs

[logs-64-0.txt](https://github.com/airbytehq/airbyte/files/7554547/logs-64-0.txt)
<details>
<summary>LOG</summary> 

</details>

## Steps to Reproduce
1. Create Google Analytics source
2. Add custom report JSON above

## Are you willing to submit a PR?
no

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
#8049 - Fix issue with custom error
#7850 - Simplify configuration

## How
When the connector gets a custom report which has unknown metric(s) or dimension(s) and API returns an error with `400 code`, the connector ignores an error with `400 code` to finish successfully sync and inform the user about an error in logs with an error message.

Changed order and name for some values. Deleted `window_in_days` and added default value as 90 days.

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated 
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</p>
</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py [hunk 1]
```diff
@@ -119,6 +119,16 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
         if next_page:
             return {"pageToken": next_page}
 
+    def should_retry(self, response: requests.Response) -> bool:
+        if response.status_code == 400:
+            self.logger.info(f"{response.json()['error']['message']}")
+            self.raise_on_http_errors = False
+
```
Comment (antixar) [inline airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py:126]: Why here? default value of the property raise_on_http_errors is True. And by idea CDK should generate an exception before. Please add a code comment about this and its reason

**Review by antixar — COMMENTED** [review]
Could you add an unit test for this case?

File: airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py (hunk 1 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py]: Updated.

**Review by antixar — APPROVED** [review]
Looks good!
Please merge and publish your changes. Don't forget to update the PR number in the changelog

**Pull request merged**